internal/transactionsubmission/README.md

```diff
@@ -1,10 +1,24 @@
+The Transaction Submission Service (TSS) allows for transaction submission at scale to the Stellar network from a single source/distribution account, which is made possible through the use of [channel accounts](https://developers.stellar.org/docs/encyclopedia/channel-accounts) spawned from said distribution account. In the context of SDP, the TSS is responsible for submitting transactions to the network for each respective payment read from a shared database table that is "queued" by the SDP. It works seamlessly by way of graceful error handling and transaction retries alongside mechanisms such as transaction fee bumps and adjustment of the rate at which new transactions are polled when certain errors are encountered.
 
-The SDP will directly 'queue' transactions (create transactions in the database) and the Transaction Submission Service will read these transactions and submit them to the Stellar Network.
+Use of the Transaction Submission Service requires at least a single of the aforementioned channel accounts to be seeded in storage in advanced. We provide a set of CLI's that can be used to create, delete, and change the number of channel accounts the service manages with ease. To learn how to use these commands, please refer to the [Channel Accounts Management](#channel-accounts-management) section below.
 
-The Transaction Submission Service requires channel accounts to be seeded in storage in advanced. To learn how to fulfill this prerequisite, please refer to the [Channel Accounts Management](#channel-accounts-management) section below.
+## Distribution Account
+Though the transaction submitter and channel account management commands allow the distribution private key to be specified upfront through the `distribution-seed` flag, we recommend users to configure it through the environment variable `DISTRIBUTION_SEED` in a one and done way to mitigate risks of exposing this information. For the distribution seed to be valid, the account associated with it must contain a balance of the asset that is specified by the payment/transaction records.
+
+## TSS Flow
+![transaction_orchestration](./docs/images/tss_tx_flow.png)
+
+Running the transaction submitter command [detailed below](#transaction-submitter) spawns a worker that continously polls the database for any 'queued' payments that have not yet reached some terminal state at a configured polling interval. 
+
+When such a payment is found, the worker attempts to look for any free channel accounts and locks the payment and account records to the same ledger number as a bundle so to prevent the occurence of race conditions that gurantee failed transactions (i.e. attempting to submit multiple per channel account concurrently resulting in `tx_bad_seq` errors, etc...), then kicks off a job for the payment to create, sign, and broadcast a transaction via Horizon.
```

Comment:
I would not use the term broadcast here. The TSS is a client that submits the transaction to an instance of Horizon+Core, and upon being ingested, this transaction will then be broadcasted to the network by the Core instance.

```suggestion
When such a payment is found, the worker attempts to look for any free channel accounts and locks the payment and account records to the same ledger number as a bundle so to prevent the occurence of race conditions that gurantee failed transactions (i.e. attempting to submit multiple per channel account concurrently resulting in `tx_bad_seq` errors, etc...), then kicks off a job for the payment to create, sign, and submit a transaction via Horizon.
```